designer/server/src/main/scala/pl/touk/nussknacker/ui/Error.scala

```diff
@@ -0,0 +1,31 @@
+package pl.touk.nussknacker.ui
+
+import akka.http.scaladsl.model.{StatusCode, StatusCodes}
+
+
+object Error {
+
+  type XError[A] = Either[Error, A]
+
+}
+
+trait Error {
```

Comment:
If that is the only place where we extend `trait Error` then we should make it sealed